PixelHunter1995/Polygon.cs

```diff
@@ -198,5 +199,43 @@ public override string ToString()
         {
             return String.Join<Coord>(", ", vertices);
         }
+
+        public void Draw(GraphicsDeviceManager graphics)
+        {
+            if (!GlobalSettings.Instance.Debug)
+            {
+                return;
+            }
+
+            VertexPositionColor[] drawVertices = new VertexPositionColor[vertices.Count];
+            short[] drawIndices = new short[vertices.Count * 2];
+            for (int i = 0; i < vertices.Count; i++)
+            {
+                Coord coord = vertices[i];
+                float x = (coord.X / graphics.PreferredBackBufferWidth) * 2.0f - 1.0f;
+                float y = -(coord.Y / graphics.PreferredBackBufferHeight) * 2.0f + 1.0f;
```

Comment:
Might be worth making a note about how the coordinate system is defined for quicker parsing when unused to the code

Reply:
Fixed!